src/themes/mainTheme.ts

```diff
@@ -75,6 +88,16 @@ const theme = createMuiTheme({
     background: {
       default: Color.Background,
     },
+    section: {
+      light: BackgroundColors.BlueLight,
+      main: BackgroundColors.Blue,
+      dark: BackgroundColors.BlueDark,
+    },
+    sectionMega: {
+      light: BackgroundColors.BlueMegaLight,
+      main: BackgroundColors.Blue,
+      dark: BackgroundColors.Black,
+    },
```

Comment:
syns det er litt rart å ha `primary`, `section` og `sectionMega` som alle egentlig er mye av samme greia, og faktisk har identisk `main`.

hadde det vært enklere å bare lage en egen BackgroundColor, type slik: https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/colors/blue.d.ts

så bruker vi bare den for å angi bakgrunner uten å måtte gå via theme?

Reply:
et objekt av typen `PaletteColor ` tar kun `light, main, dark` som variasjoner. Jeg synes det er ganske logisk å ha to "varianter" av `section` når det ikke er mulig med fem. Da har vi et objekt som er standard (`section`), og et objekt som er ekstrem (`sectionMega`). Det er også vanlig å navngi farger slik i designverdenen. Ideelt sett skulle vi hatt muligheten til å ha fem variasjoner av et egendefinert objekt, men har ikke funnet noen måte å gjøre det på. 

Er veldig lite fan av å kalle en blåfarge `blue[50], blue[200]` osv. Det sier meg ingenting om hvilken type farge det er. Det eneste jeg vet er at `blue[50]` er lysere enn `blue[200]`, men jeg vet ikke om det er designerens `megaLight` eller `light`. 

Kan se om jeg finner noen andre alternativ da. Kanskje dette er en kandidat for å bruke typen `any` 😬

Reply:
Nå bruker vi faktisk ikke primary.light eller primary.dark til noe, mens primary.main = section.main = sectionMega.main, så vi kan flytte light og dark section til primary.
Etter å ha bladd enda mer i theme objektet syns jeg kanskje light og dark sectionMega passer best som en utvidelse av `palette.background`, da blir ingen farger duplisert heller

Reply:
se på den siste endringen min nå.

Reply:
liker den bedre og kan leve med den ja :) men ser ikke behovet for både primary og section så lenge de er like, så kunne evt flytta alt til primary

Reply:
er det riktig av oss å endre typen til `primary` da? det er jo en fastsatt greie?

Reply:
hm jo sant, vi vil kanskje ikke nødvendigvis endre bakgrunnene sammen med knappefarger nei... 🤔 pr nå er det vel egentlig bare knapper som er andrevalg som er primary.main da, så jeg syns det er litt rart at primary blir en av de minst brukte, men vi kan prøve det sånn så får vi se etter hvert